spec/acceptance/pyvenv_spec.rb

```diff
@@ -143,4 +143,44 @@ class { 'python':
       apply_manifest(pp, catch_changes: true)
     end
   end
+  context 'with versioned minimal python::pip and without systempkgs' do
```

Comment:
I think this hints that rubocop-rspec is not running. There should be a newline separating them.

Reply:
updated it